test/webdav_test.dart

```diff
@@ -8,45 +8,45 @@ import 'config.dart';
 
 void main() {
   final client = getClient();
```

Comment:
For the WebDav test we probably should perform the login operation with email+password instead of username. This way we can simulate the username != login name behavior.

Reply:
I think a separate test makes more sense